### PR TITLE
Readme tweak

### PR DESCRIPTION
**Pull request by RussKie**
You can check out the updated readme here - https://github.com/RussKie/gitextensions/tree/Readme_tweak

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,16 +116,6 @@ title="See more information on Transifex.com"><img src="https://ds0k0en9abmn1.cl
...
-* Source code: [github.com/gitextensions/gitextensions](https://github.com/gitextensions/gitextensions)
-* Online manual: [git-extensions-documentation.readthedocs.org](https://git-extensions-documentation.readthedocs.org/en/latest/)
-* Issue tracker: [github.com/gitextensions/gitextensions/issues](https://github.com/gitextensions/gitextensions/issues)
-* Wiki: [github.com/gitextensions/gitextensions/wiki](https://github.com/gitextensions/gitextensions/wiki)
-* Gitter chat: [gitter.im/gitextensions/gitextensions](https://gitter.im/gitextensions/gitextensions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-* Mailing list: [groups.google.com/group/gitextensions](https://groups.google.com/group/gitextensions)
-
 # Shoutouts
 
 * We thank JetBRAINS for [ReSharper](https://www.jetbrains.com/resharper/) licenses.
```
Comment (pmiossec) [inline README.md]: A little tweak that I find could be done. Put contributors first, and after ReSharper and icons.

Comment (RussKie) [reply README.md]: Good idea, thank you

File: README.md [hunk 2]
```diff
@@ -3,19 +3,18 @@
 # Git Extensions
 
 Git Extensions is a standalone UI tool for managing git repositories.
-
 It also integrates with Windows Explorer and Microsoft Visual Studio (2015/2017).
 
 
 <a href="https://github.com/gitextensions/gitextensions/releases/latest" rel="nofollow" style="vertical-align: -webkit-baseline-middle;"><a href="#backers" alt="sponsors on Open Collective"><img src="https://opencollective.com/gitextensions/backers/badge.svg" /></a> <a href="#sponsors" alt="Sponsors on Open Collective"><img src="https://opencollective.com/gitextensions/sponsors/badge.svg" /></a> <img src="https://img.shields.io/github/downloads/gitextensions/gitextensions/total.svg?maxAge=86400" alt="GitHub Releases (latest)"></a> [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/gitextensions/gitextensions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (gerhardol) [inline README.md]: I suggest moving backers, sponsors last in row

File: README.md [hunk 3]
```diff
@@ -34,14 +33,13 @@ For more details please refer to [versions comparison](https://github.com/gitext
   <tr>
     <td>Development</td>
     <td>MS VS 2017 v15.7+, C#7.3<br/>
-        (C++ including Visual C++ ATL for x86 and x64 for installer)
+        VC++ (inc. ATL for x86/x64 for installer)
     </td>
     <td>No official support</td>
```
Comment (gerhardol) [inline README.md]: I suggest removing Mono column completely for 3.0

Comment (RussKie) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (RussKie) [reply README.md]: done

**Pull request merged**